teams/teams-ps/teams/Get-CsTeamsMeetingTemplatePermissionPolicy.md

```diff
@@ -0,0 +1,117 @@
+---
+Module Name: MicrosoftTeams
+title: Get-CsTeamsMeetingTemplatePermissionPolicy
+author: boboPD
+ms.author: pradas
+online version: https://learn.microsoft.com/powershell/module/teams/Get-CsTeamsMeetingTemplatePermissionPolicy
+schema: 2.0.0
+---
+
+# Get-CsTeamsMeetingTemplatePermissionPolicy
+
+## SYNOPSIS
+Fetches the TeamsMeetingTemplatePermissionPolicy. This policy can be used to hide meeting templates from users and groups.
+
+## SYNTAX
+```powershell
+Get-CsTeamsMeetingTemplatePermissionPolicy [[-Identity] <string>]  [<CommonParameters>]
+
+Get-CsTeamsMeetingTemplatePermissionPolicy [-Filter <string>]  [<CommonParameters>]
+```
```

Comment:
For different parameter sets you should use the following format:
![image](https://github.com/MicrosoftDocs/office-docs-powershell/assets/33589238/74c5bc05-8b48-4690-8f42-90391080928b)